popgen/dunn.go

```diff
@@ -58,15 +53,11 @@ func FindMaxIntra(subFa []*fasta.Fasta, g *Group, b *bed.Bed) int {
 //FindMinInter is a helper function of Dunn that calculates the minimum pairwise sequence distance between two sequences of a multiFa alignment that are part of different groups.
 func FindMinInter(g []*Group, subFa []*fasta.Fasta) int {
 	var answer int = math.MaxInt64
-	var group1index, group2index int
+	faMap := fasta.ToMap(subFa)
 	for i := 0; i < len(g[0].Members); i++ {
 		for j := 0; j < len(g[1].Members); j++ {
-			group1index = fasta.FindFaIndex(subFa, g[0].Members[i])
-			group2index = fasta.FindFaIndex(subFa, g[1].Members[j])
-			if group1index != -1 && group2index != -1 {
-				answer = numbers.Min(answer, dna.Dist(subFa[group1index].Seq, subFa[group2index].Seq))
-			}
+			answer = numbers.Min(answer, dna.Dist(faMap[g[0].Members[i]], faMap[g[1].Members[j]]))
```

Comment:
Same comment about seq names not being found in the map (@rimangan)

Reply:
I have added a check and panic if fastas not found. Riley, let me know if this should not panic.